recipes/docstring-parser/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "docstring-parser" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f91d48ea2e4ad04a101f3aa24767f8475545acc93af1b3b14c617733d40c086
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - docstring_parser
+
+about:
+  home: https://github.com/rr-/docstring_parser
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'Parse Python docstrings in reST and Google format '
+  description: |
+    Parse Python docstrings. Currently support ReST, Google, and Numpydoc-style
+    docstrings.
+  dev_url: https://github.com/rr-/docstring_parser
+
+extra:
+  recipe-maintainers:
+    - Chilipp
+    - rr-
```

Comment:
@rr- are you willing to be a maintainer?

Reply:
Yes